packages/react/src/styles/_typography.scss

```diff
@@ -0,0 +1,84 @@
+@import './layout';
+
+@mixin title-base {
+  font-family: HelsinkiGrotesk;
+}
+
+
```

Comment:
Extra line here and after this new lines are missing between mixins, it would improve readability to add them 👍